src/plugins/sync-plugin.js

```diff
@@ -780,7 +797,7 @@ const createTypeFromTextOrElementNode = (node, mapping) =>
 const isObject = (val) => typeof val === 'object' && val !== null
 
 const equalAttrs = (pattrs, yattrs) => {
-  const keys = Object.keys(pattrs).filter((key) => pattrs[key] !== null)
+  const keys = Object.keys(pattrs).filter((key) => pattrs[key] !== null && !key.startsWith('yelement_mark_'))
```

Comment:
Why are you filtering `yelement_mark_` here in the attributes that you get from the prosemirror node? If there is a reason, why are you not filtering in the next line when you are comparing?